src/components/common/CustomCalendar.tsx

```diff
@@ -109,25 +123,33 @@ const CustomCalendar: FC<CustomCalendarProps> = (props) => {
         />
         )
         } />
+      <CustomToast
+        onHide={() => setShowFutureDateToast(false)}
+        showToast={showFutureDateToast}
+        severity="warn"
+        detail="Please select a date and time on or before the current moment."
+        position="top-center"
+        life={3000}
+      />
     </div>
   )
 }
 
 const ManualInputTime: FC<ManualInputTimeProps> = ({ hour, minute, handleHourChange, handleMinuteChange, ampm, toggleAmPmChange }) => {
   return (
     <div className="flex justify-evenly w-[12rem] mx-auto items-center">
-      <InputNumber value={hour} step={1} onValueChange={(e) => handleHourChange(e)} showButtons buttonLayout="vertical" min={0} max={12} /> :
-      <InputNumber value={minute} onValueChange={(e) => handleMinuteChange(e)} showButtons buttonLayout="vertical" step={1} min={0} max={59} />:
+      <CustomInputNumber value={hour < 1 ? 12 : hour} step={1} onValueChange={(e) => handleHourChange(e)} showButtons buttonLayout="vertical" min={0} max={12} prefix={hour && hour < 10 && "0"} /> :
```

Comment:
Such a long line.
On save, it should have aligned into multiple lines.
Can you check if your linter is working as expected ?

Reply:
I checked prettier settings. I couldn't find a solution. It wont automatically update for me. For now, I  fixed this